src/routes/healthCheck.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  res.json({ status: "Server is running" });
+});
+
+/**
+ * @swagger
+ * /health-check:
+ *    get: 
+ *      tags: 
+ *        - health
+ *      summary: Server health check
+ *      description: This api is used to check server health
+ *      responses: 
+ *        200:
+ *          description: Server is running       
+ */
```

Comment:
It would be better to describe error responses here too.

Reply:
I've added responses to docs. May I merge?

Reply:
Yes. Approved.